app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -330,6 +346,28 @@ class HomeFragment : Fragment() {
             )
         }
 
+        requireContext().settings().showUnifiedSearchFeature.let {
+            binding.searchSelector.isVisible = it
+            binding.searchEngineIcon.isGone = it
+        }
+
+        binding.searchSelector.apply {
+            setOnClickListener {
+                val orientation = if (context.settings().shouldUseBottomToolbar) {
+                    Orientation.UP
+                } else {
+                    Orientation.DOWN
+                }
+
+                UnifiedSearch.searchMenuTapped.record(mozilla.telemetry.glean.private.NoExtras())
```

Comment:
```suggestion
                UnifiedSearch.searchMenuTapped.record(NoExtras())
```

Can we clean up the full import here?